robot.py

```diff
@@ -11,7 +12,9 @@
 
 import wpilib
 import commands2
-import commands2.cmd
+#import commands2.cmd
```

Comment:
If we no longer need `commands2.cmd`, we should just delete the line.  In general, it's not a great idea to check in files with commented-out code.  Either the commented lines are no longer needed, or we're not sure how everything works which means the code probably isn't finished.